src/workspace/mod.rs

```diff
@@ -2748,6 +2752,7 @@ mod test {
             include: Some(vec!["rule1".to_string(), "rule2".to_string()]),
             exclude: Some(vec![])
           },
+          plugins: vec!["jsr:@deno/test-plugin2".to_string()],
```

Comment:
We need to decide if plugins should be inherited from the workspace root

Reply:
I think so.

Reply:
One issue with that though, is how does someone say they don't want a plugin used in a member that's found at the root? Maybe we should support adding `!` before a plugin in the future?

Reply:
I think that's reasonable - I'll wait for @marvinhagemeister opinion before merging this one.

Reply:
Inheriting sounds reasonable. We could add `!` in the future, but I'd kinda defer that until someone asks for it.